Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -329,6 +335,21 @@ void GcodeSuite::G28(const bool always_home_all) {
         delayed_move_time = 0;
         active_extruder_parked = true;
 
+        if (IDEX_saved_mode >= DXC_DUPLICATION_MODE) {
+          enqueue_and_echo_commands_P("M605 S1\nT0");
+          char idexbuffer[20];
+          sprintf_P(idexbuffer, PSTR("M605 S2 X%i"), DEFAULT_DUPLICATION_X_OFFSET);
+          enqueue_and_echo_commands_P(idexbuffer);
```

Comment:
In that case, looking over queueing options, is enqueue_and_echo_command_now() more functional here?

Reply:
You have to be careful with `enqueue_and_echo_command_now` because if the buffer is full Marlin will recurse back into the menu handler when `enqueue_and_echo_command_now` calls `idle()`.